assignment-2/strava-god/src/app/material/material.module.ts

```diff
@@ -0,0 +1,51 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import * as Material from "@angular/material";
```

Comment:
Dont import everything from the material module. Make import for every component